src/lib/formatRoomType.ts

```diff
@@ -0,0 +1,24 @@
+import { RoomType } from "@prisma/client";
+
+export const formatRoomType = (roomType: RoomType): string => {
+  switch (roomType) {
+    default:
+    // case RoomType.INVALID:
+    //   return "Invalid room type";
+
```

Comment:
```suggestion
     case RoomType.INVALID:
       return "Invalid room type";
```

can be commented in again as you added the `INVALID` type to your prisma schema

Reply:
✅